server/middleware/prepaids.coffee

```diff
@@ -214,6 +304,7 @@ module.exports =
       continue unless prepaid.creator
       userPrepaidsMap[prepaid.creator.valueOf()] ?= []
       userPrepaidsMap[prepaid.creator.valueOf()].push(prepaid)
+      # NOTE: May not correctly account for shared licenses
```

Comment:
I'd say just add any joiners to the list of userIDs. It's only going to add more to the userIDs and may get some additional trial requests.